nuget/determine-pr-version-base.yml

```diff
@@ -0,0 +1,77 @@
+parameters:
```

Comment:
Instead of calling it base, why not move it to `/determine-pr-version.yml` and refer to that one in `/<package manager>/determine-pr-version.yml`?

Reply:
Moving to the root folder? Hmm, isn't that a bit 'scattered' than? Do we have a 'base' folder or something. Or a templates folder for the templates 😅 .

Reply:
I'd go with the root because `base` says nothing and a templates folder would be confusing indeed

Reply:
Ok, yes, will do!